lib/miq_automation_engine/engine/miq_ae_service/miq_ae_service_rbac.rb

```diff
@@ -0,0 +1,33 @@
+module MiqAeMethodService
+  module MiqAeServiceRbac
+    extend ActiveSupport::Concern
+
+    module ClassMethods
+      def find_ar_object_by_id(id)
+        Rbac.search(:class => model, :targets => [id],
+                    :results_format => :objects).first.first
+      end
+
+      def all
+        wrap_results(Rbac.search(:class => model, :results_format => :objects).first)
```

Comment:
Please pass user into `Rbac.search` - relying upon the global `User.current_user` is more safe (no guarantee that will be set)